docs/REST API/rest-api.md

```diff
@@ -4233,7 +4247,16 @@ List all configuration profiles for macOS and Windows hosts enrolled to Fleet's
       "platform": "windows",
       "created_at": "2023-04-31T00:00:00Z",
       "updated_at": "2023-04-31T00:00:00Z",
-      "checksum": "aCLemVr)"
+      "checksum": "aCLemVr)",
+      "labels": [
```

Comment:
@noahtalerman @marko-lisica I think the `labels` array should also be included in 
`GET /api/v1/fleet/mdm/profiles/:profile_uuid` (i.e. get metadata for a specific profile), right?

Reply:
Also, I think we should add `id` so the label ID is present in the payload and users can use it to make other label-related API calls from this response? E.g. https://fleetdm.com/docs/rest-api/rest-api#get-label.

Reply:
@mna Good catch. I added `labels` array. I skipped `id` just to be consistent with `/profiles` endpoint and the user needs to specify a name only in order to assign a label to a profile.